app/models/mpc_offender.rb

```diff
@@ -189,6 +189,11 @@ def prison_timeline
     @prison_timeline ||= HmppsApi::PrisonTimelineApi.get_prison_timeline(offender_no)
   end
 
+  # Calling a helper from a model? Required to make sorting work in unallocated view
+  def additional_information
+    ApplicationController.helpers.additional_information(self)
+  end
+
```

Comment:
I'm not happy with this. I know you are not either.

Why not move `additional_information` into MpcOffender then call it from the helper instead of the other way around?